content/ideas/hackathon-project-descriptions.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Hackathon Project Descriptions"
+date: "2018-09-14"
+---
+
+### [Project #1: Audible incoming traffic warnings @ intersections]({{< relref "/hackathon/prd/audible-incoming-traffic-warnings.md" >}})
+
+An occupancy sensor (pressure sensor, camera, etc.) in the street that lets people know about someone approaching on a bicycle or car if they're preparing to cross the street and / or currently in the street. [Read more]({{< relref "/hackathon/prd/audible-incoming-traffic-warnings.md" >}})
+
+### [Project #2: Tranquil Refuge]({{< relref "/hackathon/prd/tranquil-refuge.md" >}})
+
+A place that people that need a physical or sensory reprieve from the environment can go to rest and recharge. Potential to give priority access to people with accessibility needs. [Read more]({{< relref "/hackathon/prd/tranquil-refuge.md" >}})

```

Comment:
I think this would sound better as 'A place for people that....'

Reply:
@sepidehshahi What do you think? Can we change the wording?